app/controllers/articles.js

```diff
@@ -30,116 +31,169 @@ exports.load = function (req, res, next, id){
 exports.index = function (req, res){
   var page = (req.params.page > 0 ? req.params.page : 1) - 1;
   var perPage = 30;
-
   var options = {
-          perPage: perPage,
-          page: page
-        };
+    perPage: perPage,
+    page: page,
+    filters: {}
+  };
+  req.query.title ? options.filters.title = {'$regex': req.query.title} : "";
```

Comment:
What do you want add to ```options.filters.title``` if req.query.title does not exist? Do you want that ```options.filters.title``` will be equal "" or undefined? Now it is undefined.